### DISCUSSION TITLE
How to use open source to build a resume

### DISCUSSION THREAD

**Discussion by adiati98** [Virtual-Coffee/virtualcoffee.io#529] [Open Source]
Hi Friends,

@dominicduffin1 and I are working on updating our [Open Source Resources](https://virtualcoffee.io/resources/open-source/) page on the website.

It's not a secret that contributing to open sources can help devs to build their resume and even lead them to their job. 
So, we want to have a section about how to use open source to build a resume.

If you have any experience yourself or know any good resources in using open source to build a resume, we would love to hear about them!

Please drop them in the discussion here.

Thank you in advance! 😄

**Comment by horacioh** [comment]
Open source contributions involves a lot of skills that many companies strive for. not only helps you get great jobs, it also helps you become a better programmer and communicator. IMHO there's no better way!

some resources I know from the top of my head:

- https://kentcdodds.com/blog/introducing-how-to-contribute-to-open-source
- https://kentcdodds.com/blog/open-source-stamina
- https://kentcdodds.com/blog/what-open-source-project-should-i-contribute-to
- https://twitter.com/jsjoeio (follow him, is a MUST!)
- https://www.freecodecamp.org/news/i-made-my-first-open-source-contribution-within-200-days-and-how-you-can-too-4d5bdbd63fad

If I find more, I will share it here!

**Reply by adiati98** [reply]
Thank you, @horacioh! 😄

**Comment by rokuweb** [comment]
Open source has no doubt become part of our lives as creators and the benefits of contributing to open source cannot be over emphasized.
Check out this article on how to contribute to open source;
https://logique.hashnode.dev/how-to-contribute-to-open-source

**Reply by adiati98** [reply]
Thank you, @logiquebox ! 🙏😄

**Comment by nikhilkuyya** [comment]
TwiloQuest: to get understanding of opensource
https://www.twilio.com/quest
https://www.twilio.com/quest/learn/open-source?utm_source=hacktoberfest2019&utm_campaign=hacktoberfest2019
[Video Explaining TwilioQuest](https://youtu.be/Nxja-95Z2zc)

**Reply by adiati98** [reply]
Thank you, @nikhilkuyya! 🙏😄
